libs/dh-app/core/feature-shell/src/lib/dh-app-core-feature-shell.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+
+import { DhAppCoreFeatureShellRoutingModule } from './dh-app-core-feature-shell-routing.module';
+import { ShellModule } from './shell/shell.module';
+
+@NgModule({
+  imports: [ShellModule, DhAppCoreFeatureShellRoutingModule],
+})
+export class DhAppCoreFeatureShellModule {}
```

Comment:
suggestion: a feature shell module contains root level routes, so we use `RouterModule.forRoot`
suggestion: make the feature shell idiomatic and simpler by deleting the separate routing module

```suggestion
import { NgModule } from '@angular/core';
import { RouterModule, Routes } from '@angular/router';

import { ShellComponent } from './shell/shell.component';
import { ShellModule } from './shell/shell.module';

const routes: Routes = [
  {
    path: '',
    component: ShellComponent,
    children: [
      // Lazy feature routes
    ],
  },
];

@NgModule({
  exports: [RouterModule],
  imports: [RouterModule.forRoot(routes), ShellModule],
})
export class DhAppCoreFeatureShellModule {}
```